### PR TITLE
Bel update 02.08

### PR DESCRIPTION
**Pull request by antikruk**

### CODE REVIEW

**Comment by Skif-off** [comment]
`crowdin.yml` is not needed.

Can you also try
https://github.com/doublecmd/doublecmd/tree/master/plugins/wcx/unrar/language
https://github.com/doublecmd/doublecmd/tree/master/plugins/wcx/zip/language
https://github.com/doublecmd/doublecmd/blob/master/plugins/wdx/audioinfo/audioinfo.lng
https://github.com/doublecmd/doublecmd/blob/master/plugins/wfx/ftp/language/ftp.be.po#L34
?

P.S. Maybe this https://github.com/j2969719/doublecmd-plugins/tree/master/plugins/po will be interesting for you.

**Comment by antikruk** [comment]
Ok. I'll watch it later. Can you create a project on Crowdin or Weblate? Didn't think about it?

File: language/doublecmd.be.po [hunk 1]
```diff
@@ -3,17 +3,19 @@ msgstr ""
...
 "POT-Creation-Date: 2008-01-17 23:08+0300\n"
-"PO-Revision-Date: 2020-05-30 08:13\n"
+"PO-Revision-Date: 2021-08-02 10:35\n"
 "Last-Translator: Full Name <email@address>\n"
 "Language-Team: Belarusian\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Native-Language: Беларуская\n"
+"X-Native-Language: English\n"
```
Comment (alexx2000) [inline language/doublecmd.be.po:12]: Why you changed it? It should be language name at native language.

Comment (antikruk) [reply language/doublecmd.be.po]: Sorry, auto

**Pull request merged**